classes/ActionScheduler_Store.php

```diff
@@ -120,12 +127,37 @@
 	 */
 	abstract public function get_status( $action_id );
 
+	/**
+	 * @return mixed
+	 */
+	abstract public function get_claim_id( $action_id );
+
 	/**
 	 * @param string $claim_id
 	 * @return array
 	 */
 	abstract public function find_actions_by_claim_id( $claim_id );
 
+	protected function validate_sql_comparator( $comp ) {
+		if ( in_array($comp, array('!=', '>', '>=', '<', '<=', '=')) ) {
```

Comment:
This needs some formatting adjustments to meet with WP standards.

I think it would be preferable to use a key-based array, but it also looks like this will typically only be used a couple times during a given request, so the performance benefit would be minimal. Meaning, it's also OK to leave as-is 🙂